res/css/_spacing.pcss

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 /* 1rem :: 10px */
 
+$spacing-2: 2px;
```

Comment:
I thought the whole point of the `$spacing` variables was to encourage people not to use values which are not a multiple of 4?

Reply:
1. I was not aware that we only accept multiple of 4 (the code style doesn't mention it https://github.com/vector-im/element-web/blob/develop/code_style.md#stylesheets-pcss--postcss--plugins)
2. The expected design was with a 2px spacing 🧐

Reply:
I realise those things, but my question is, what is the point of the `$spacing`, if we're just going to introduce new entries whenever we need one? `$spacing-2` is a lot longer to type than `2px`. I just don't get it.

I don't mean to pick on you @florianduros - I think it's a question for the design team: @gsouquet can you give us any guidance on use the `$spacing` variables?

Reply:
https://vector-im.github.io/compound/?path=/story/styles-styles--spacing